setup.cfg

```diff
@@ -44,3 +45,4 @@ python_requires = >=3.7
 
 [options.extras_require]
 all =
+    dwave-gate==0.1.0
```

Comment:
I was thinking something more specific than `all`, since we don't include drivers and the inspector with `all`.. Perhaps `gate`? Have no strong opinion on this one.

Reply:
I also have no strong opinion. `all` seemed safe but happy to do it in others as well.